### PR TITLE
Show course names in semester view

### PR DESCRIPTION
**Pull request by cubetastic33**
NP-19
+ Show the course names in the semester view
+ Show the warning and lock icons next to the course ID
+ Show an ellipse when the course is hovered upon
+ Show the menu only when the ellipse is clicked
+ Only show the option to dismiss pre-reqs when there are prereq warnings

image

### CODE REVIEW

File: src/components/planner/Tiles/SemesterCourseItem.tsx [hunk 1]
```diff
@@ -143,26 +123,47 @@ export const MemoizedSemesterCourseItem = React.memo(
...
-                  <span className="text-[#FBBF24]">
+                  <span className="ml-1 text-[#FBBF24]">
                     <FilledWarningIcon />
                   </span>
                 )}
+                {course.locked && <LockIcon className="ml-1" />}
               </span>
-
-              {course.locked && <LockIcon />}
+              <span className="w-72 truncate text-sm">{title}</span>
```
Comment (ZzRanger) [inline src/components/planner/Tiles/SemesterCourseItem.tsx:136]: This is causing issues in the semesters haha.

Screenshot 2023-04-11 at 3 30 23 PM

**Review by ZzRanger — CHANGES_REQUESTED** [review]
Changes look great I really like the hamburger button!! We just need to fix the UI a bit as on laptop screens the summer semesters are cut off. Otherwise great work!

File: src/components/planner/Tiles/SemesterCourseItem.tsx (hunk 1 above)
Comment (cubetastic33) [reply src/components/planner/Tiles/SemesterCourseItem.tsx]: Made some changes; should look like this now on small screens

Screen Shot 2023-04-11 at 18 05 17

**Review by akevinge — APPROVED** [review]
Looks good! This is super dope to have :)

**Pull request merged**